src/App.js

```diff
@@ -1,6 +1,9 @@
 import "./index.css";
 import headerLogo from "./images/logo_place_header.svg";
+import footerLogo from "./images/logo_place_footer.svg";
 import headerImage from "./images/two_again.png";
+import feynman from "./images/feynman.png";
+import kaufman_triangle from "./images/kaufman-triangle.svg";
```

Comment:
Why did you use camelCase for `footerLogo` and `headerImage` but switch to snake_case? Use camelCase instead

Reply:
done